src/pages/resultsView/fusion/ResultViewFusionMapperDataStore.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2018 The Hyve B.V.
+ * This code is licensed under the GNU Affero General Public License (AGPL),
+ * version 3, or (at your option) any later version.
+ *
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **/
+
+import * as _ from 'lodash';
+import { StructuralVariant } from "../../../shared/api/generated/CBioPortalAPI";
+import { action, observable, computed } from "mobx";
+import Immutable from "seamless-immutable";
+import { SimpleLazyMobXTableApplicationDataStore } from '../../../shared/lib/ILazyMobXTableApplicationDataStore';
+
+type PositionAttr = { [position: string]: boolean };
+type ImmutablePositionAttr = PositionAttr & Immutable.ImmutableObject<PositionAttr>;
+
+export default class FusionMapperDataStore extends SimpleLazyMobXTableApplicationDataStore<StructuralVariant[]> {
+    @observable.ref private selectedPositions: ImmutablePositionAttr;
```

Comment:
instead of using immutables, a good way (and more typical in our codebase) of doing this is using mobx `observable.map`

Reply:
oh I see now that you just copied the way we do it in MutationMapperDataStore. Okay I think you should just leave it then, sorry.